docs/include/filter.asciidoc

```diff
@@ -0,0 +1,171 @@
+[id="plugins-{type}s-common-options"]
+==== Common Options
+
+The following configuration options are supported by all filter plugins:
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Setting |Input type|Required
+| <<plugins-{type}s-{plugin}-add_field>> |<<hash,hash>>|No
+| <<plugins-{type}s-{plugin}-add_tag>> |<<array,array>>|No
+| <<plugins-{type}s-{plugin}-enable_metric>> |<<boolean,boolean>>|No
```

Comment:
Do we need enable_metric and id documented here if they are already documented in the base plugin docs?

Or to ask another way, do we need base plugin docs if we  include the base config settings in the plugin types?